src/main/python/rlbot/matchconfig/match_config.py

```diff
@@ -66,22 +66,33 @@ def __init__(self):
-        mutator_settings.game_speed_option = game_speed_mutator_types.index(self.game_speed)
-        mutator_settings.ball_max_speed_option = ball_max_speed_mutator_types.index(self.ball_max_speed)
-        mutator_settings.ball_type_option = ball_type_mutator_types.index(self.ball_type)
-        mutator_settings.ball_weight_option = ball_weight_mutator_types.index(self.ball_weight)
-        mutator_settings.ball_size_option = ball_size_mutator_types.index(self.ball_size)
-        mutator_settings.ball_bounciness_option = ball_bounciness_mutator_types.index(self.ball_bounciness)
-        mutator_settings.boost_amount_option = boost_amount_mutator_types.index(self.boost_amount)
-        mutator_settings.rumble_option = rumble_mutator_types.index(self.rumble)
-        mutator_settings.boost_strength_option = boost_strength_mutator_types.index(self.boost_strength)
-        mutator_settings.gravity_option = gravity_mutator_types.index(self.gravity)
-        mutator_settings.demolish_option = demolish_mutator_types.index(self.demolish)
-        mutator_settings.respawn_time_option = respawn_time_mutator_types.index(self.respawn_time)
+        mutator_settings.match_length = self.index_or_zero(match_length_types, self.match_length)
+        mutator_settings.max_score = self.index_or_zero(max_score_types, self.max_score)
+        mutator_settings.overtime_option = self.index_or_zero(overtime_mutator_types, self.overtime)
+        mutator_settings.series_length_option = self.index_or_zero(series_length_mutator_types, self.series_length)
+        mutator_settings.game_speed_option = self.index_or_zero(game_speed_mutator_types, self.game_speed)
+        mutator_settings.ball_max_speed_option = self.index_or_zero(ball_max_speed_mutator_types, self.ball_max_speed)
+        mutator_settings.ball_type_option = self.index_or_zero(ball_type_mutator_types, self.ball_type)
+        mutator_settings.ball_weight_option = self.index_or_zero(ball_weight_mutator_types, self.ball_weight)
+        mutator_settings.ball_size_option = self.index_or_zero(ball_size_mutator_types, self.ball_size)
+        mutator_settings.ball_bounciness_option = self.index_or_zero(ball_bounciness_mutator_types, self.ball_bounciness)
+        mutator_settings.boost_amount_option = self.index_or_zero(boost_amount_mutator_types, self.boost_amount)
+        mutator_settings.rumble_option = self.index_or_zero(rumble_mutator_types, self.rumble)
+        mutator_settings.boost_strength_option = self.index_or_zero(boost_strength_mutator_types, self.boost_strength)
+        mutator_settings.gravity_option = self.index_or_zero(gravity_mutator_types, self.gravity)
+        mutator_settings.demolish_option = self.index_or_zero(demolish_mutator_types, self.demolish)
+        mutator_settings.respawn_time_option = self.index_or_zero(respawn_time_mutator_types, self.respawn_time)
+
+    @staticmethod
+    def index_or_zero(types, value):
```

Comment:
"types" seems like a tricky thing to understand when first reading this.
How about refactoring our lists to enums such that we have a better type signature. e.g.
`def index_or_zero(enum_type: Type[Enum], value: Enum) -> int:`

Changing things to enums is fine even if we have names with spaces. e.g. 
```
Station = Enum(
    value='Station',
    names=[
        ('wien_westbahnhof', 1),
        ('Wien Westbahnhof', 1),
        ('st_poelten', 2),
        ('St. Pölten', 2),
        ('linz', 3),
        ('Linz', 3),
    ]
)
```

If we are changing things to enums, we could make things even more convenient by putting the index_or_zero on a class our Enums inherit from such that the above lines look like this:
`mutator_settings.demolish_option = DemolishMutator.parse_or_default(self.demolish)`
I'd find this approach much more readable.